File: Python Code/Daily_Flash_Code_0/Week 3/28Jan/13-DailyFlash_Solutions/13-DailyFlash_Solutions/28_Jan_Solutions_Two/C++/prog5.cpp
#include<iostream>

int main() {
	int num1, num2, num3;
	std::cout << "Enter num1 : ";
	std::cin >> num1;
	std::cout << "Enter num2 : ";
	std::cin >> num2;
	std::cout << "Enter num3 : ";
	std::cin >> num3;
	if(num1 <= num2 && num1 <= num3) {
		std::cout << "Min = " << num1;
	} else if(num2 <= num3) {
		std::cout << "Min = " << num2;
	} else {
		std::cout << "Min = " << num3;
	}
	return 0;
}

File: Python Code/Daily_Flash_Code_0/Week 3/28Jan/13-DailyFlash_Solutions/13-DailyFlash_Solutions/28_Jan_Solutions_Two/Python/prog5.py
num1, num2, num3= [int(i) for i in input("Input : ").split()]
if(num1 <= num2 and num1 <= num3):
    print("Min : ", num1)
elif(num2 <= num3):
    print("Min : ", num2)
else:
    print("Min : ", num3)
